src/soter/boringssl/soter.mk

```diff
@@ -15,6 +15,10 @@
 #
 
 SOTER_SRC += $(wildcard $(CRYPTO_ENGINE)/*.c)
+SOTER_AUD_SRC += $(wildcard $(CRYPTO_ENGINE)/*.c)
+SOTER_AUD_SRC += $(wildcard $(CRYPTO_ENGINE)/*.h)
```

Comment:
It turned out that BoringSSL build for audit did not include BoringSSL code for Soter. I've added it here.